src/js/helpers/error-handler.js

```diff
@@ -2,17 +2,29 @@ import { errors } from '@/js/errors'
 import { Bus } from '@/js/helpers/event-bus'
 import log from 'loglevel'
 import i18next from 'i18next'
+import { ErrorTracker } from '@/js/helpers/error-tracker'
 
 export class ErrorHandler {
-  static process (error, translationId = '') {
-    ErrorHandler.processWithoutFeedback(error)
-    Bus.error(translationId || ErrorHandler._getTranslationId(error))
+  static process (error, translationId = '', sentryReportConfig = {}) {
+    const msg = translationId || ErrorHandler._getTranslationId(error)
+    ErrorHandler.processWithoutFeedback(error, sentryReportConfig)
+    Bus.error(msg)
   }
 
-  static processWithoutFeedback (error) {
+  static processWithoutFeedback (error, sentryReportConfig = {}) {
+    ErrorHandler.captureSentryMessage(error, sentryReportConfig)
     log.error(error)
   }
 
+  static captureSentryMessage (error, opts = {}) {
+    const { sentryReportTitle = '', skipSentryReport = false } = opts
```

Comment:
1. actually, `sentryReportTitle` replaces message, so better rename it to a something that contains `message`
2. it’s not `report` anymore, but `track`

Reply:
Renamed